### ISSUE TITLE
Virtua Cop 2 - lightgun mouse causes lag

### ISSUE THREAD

**Issue by ner00** [libretro/flycast#1082]
Whenever I shoot or reload the game freezes for about half a second, this is something that I have never seen on any other Dreamcast game that uses lightgun.

Possibly related to this one: https://github.com/libretro/flycast/issues/913

Using lightgun mode: https://i.giphy.com/media/77267GNuvcGDz2G9a5/giphy.mp4

Using controller mode: https://i.giphy.com/media/a4IRJbCzjqM6oUEvnH/giphy.mp4

**Comment by bslenul** [comment]
Yeah same here, tested on Windows 10 and on my Linux Mint VM with both the JP and the US version (from Sega Smash Pack - Volume 1). I messed with the core options a bit, different input and video drivers as well but couldn't find anything that fixes it. It works perfectly fine with Flycast standalone.
